sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Offer.java

```diff
@@ -132,4 +132,14 @@ private ObjectNode getContent() {
     private void setContent(ObjectNode offerContent) {
         BridgeInternal.setProperty(this, Constants.Properties.OFFER_CONTENT, offerContent);
     }
+
+    @Override
+    public String getString(String propertyName) {
+        return super.getString(propertyName);
```

Comment:
ditto, here and other places for the new methods, the implementation is the same as parent class why are they needed in every child class?